atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -236,10 +238,32 @@ def _make_labels(self):
 
         return labels
 
+    def get_fs_task_id(self):
+        task_id = None
+        fs_result = self.workflow.prebuild_results.get(AddFilesystemPlugin.key)
+        if fs_result is not None:
+            try:
```

Comment:
Okay, what did you have in mind here? I've been toying with this, and it seems I'm doing something wrong, since the self-tests I'm working on seem to never be able to catch an expected exception here.